include/tudat/astro/gravitation/gravityFieldModel.h

```diff
@@ -95,6 +95,17 @@ class GravityFieldModel
         return computeGravitationalAcceleration( bodyFixedPosition, gravitationalParameter_ );
     }
 
+    //! Get the gradient of the laplacian of potential.
+    /*!
+     * Returns the laplacian of the gravitational potential for the gravity field selected.
+     * \param bodyFixedPosition Position at which gradient of potential is to be determined.
+     * \return Laplacian of potential.
+     */
+    virtual double getLaplacianOfPotential ( const Eigen::Vector3d& bodyFixedPosition )
```

Comment:
Why not make this pure virtual? To prevent *every* new gravity field from having to define a Laplacian function?

Reply:
Not sure what you mean... wouldn't making it pure virtual require every new gravity field to define the function implementation?

Reply:
It would yes.

Reply:
Ah yes, sorry, I misunderstood your initial question. Here it goes:
> To prevent every new gravity field from having to define a Laplacian function?

Yes :)